framework/quota/src/main/java/org/apache/cloudstack/quota/QuotaManagerImpl.java

```diff
@@ -308,15 +303,15 @@ protected List<UsageVO> getPendingUsageRecordsForQuotaAggregation(AccountVO acco
     protected List<QuotaUsageVO> createQuotaUsagesAccordingToQuotaTariffs(AccountVO account, List<UsageVO> usageRecords,
             Map<Integer, Pair<List<QuotaTariffVO>, Boolean>> mapQuotaTariffsPerUsageType) {
         String accountToString = account.reflectionToString();
-        s_logger.info(String.format("Calculating quota usage of [%s] usage records for account [%s].", usageRecords.size(), accountToString));
+        s_logger.info(String.format("Calculating quota usage of [%s] usage records for %s.", usageRecords.size(), accountToString));
```

Comment:
minor but this means that in a search in the log the '[]' can no longer be included and false positive might pop up;
think account `dahn` and account `dahn-admin` or `dahn-south`